R/agglomerate.R

```diff
@@ -23,9 +23,11 @@
 #' @param rank a single character defining a taxonomic rank. Must be a value of
 #'   \code{taxonomyRanks()} function.
 #'
-#' @param onRankOnly \code{TRUE} or \code{FALSE}: Should information only from
+#' @param disable.taxonomy \code{TRUE} or \code{FALSE}: Should information only from
 #'   the specified rank be used or from ranks equal and above? See details.
-#'   (default: \code{onRankOnly = FALSE})
```

Comment:
Remember to add

@param onRankOnly Deprecated. Please use \code{disable.taxonomy}.

Reply:
the @param is on line 30